cmd/crc/cmd/start.go

```diff
@@ -133,3 +137,19 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func checkIfNewVersionAvailable(noUpdateCheck bool) {
+	if noUpdateCheck {
+		return
+	}
+	newVersionAvailable, newVersion, err := version.NewVersionAvailable()
+	if err != nil {
+		logging.Debugf("Error checking if a new version is available: %v", err)
+		return
+	}
+	if newVersionAvailable {
+		logging.Warnf("A new version (%s) has been published. Please download it from %s", newVersion, constants.CrcLandingPageURL)
```

Comment:
Should we put a `warn` message or a `info` message. I was under the impression that we agreed to use `info` here. 

Also how about message is like `A new version (1.0.1-dev) has been published on https://cloud.redhat.com/openshift/install/crc/installer-provisioned` so in the future when we have some kind of upgrade mechanism we can also add use `crc update` in the message without making it too big ?